### DISCUSSION TITLE
Where to report graphical bugs

### DISCUSSION THREAD

**Discussion by stonerl** [italomandara/CXPatcher#86] [General]
Since there maybe bugs related to DXVK/MoltenVK, e.g., graphical glitches, where should these bugs be reported, when using CXPatcher?

**Comment by italomandara** [comment]
Usually this is due to MoltenVK so I guess MoltenVK repo, but in order to do that it's better to do some debug and provide them some info on what feature is broken rather than the usual "Fix my game" ticket.

**Reply by italomandara** [reply]
example: https://github.com/KhronosGroup/MoltenVK/issues/1974
